### PR TITLE
Releaes 2.33.0

### PR DESCRIPTION
**Pull request by szarnekow**

### CODE REVIEW

File: xtend-website/_posts/releasenotes/2023-11-20-version-2-33-0.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title:  Xtend 2.33.0 Release Notes
+date:   2023-11-20
+categories: releasenotes
+published: false
```
Comment (cdietrich) [inline xtend-website/_posts/releasenotes/2023-11-20-version-2-33-0.md:6]: change to true

File: xtext-website/_posts/releasenotes/2023-11-20-version-2-33-0.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+As you might have recognized, the number of people contributing to Xtext on a regular basis has declined over the past years and so has the number of contributions. At the same time the amount of work for basic maintenance has stayed the same or even increased with the new release cadence of Java and the Eclipse simultaneous release. Briefly: The future maintenance of Xtext is at risk, at least in the current form and as part of the Eclipse Simrel. If you care, please join the discussion in [https://github.com/eclipse/xtext/issues/1721](https://github.com/eclipse/xtext/issues/1721).
+
+
+## Upgrades
+
+* Guava was update to 32.1.3
+* Reload4j was updated to 1.2.25
+* Classgraph was updated to 4.8.164. The bundle id in orbit has changed so you may need to adapt.
+* ASM is now on 9.6
+* MWE(2) is now on 1.20/2.16
```
Comment (cdietrich) [inline xtext-website/_posts/releasenotes/2023-11-20-version-2-33-0.md:22]: should read 1.10

File: xtext-website/documentation/350_continuous_integration.md [hunk 3]
```diff
@@ -307,6 +307,7 @@ To further speed up the p2 dependency resolution step, use the concrete build re
 
 | Xtext         | EMF           | MWE2/MWE    | Xpand       | Eclipse     | All included in |
 | ------------- | ------------- | ----------- | ----------- | ----------- | ----------- |
+| [2.33.0]({{page.upsite.xtext}}releases/2.33.0/)           | [2.36.0]({{page.upsite.eclipse}}modeling/emf/emf/builds/release/2.36.0) (2.29.0)     | [2.15.0]({{page.upsite.mwe}}releases/2.15.0/) (2.9.1) | no longer supported  | [4.20.0]({{page.upsite.eclipse}}releases/2023-12) (4.23) | [2023-12]({{page.upsite.eclipse}}releases/2023-12)|
```
Comment (cdietrich) [inline xtext-website/documentation/350_continuous_integration.md:310]: emf version needs bumping

Comment (szarnekow) [reply xtext-website/documentation/350_continuous_integration.md:310]: mwe or emf? or both?

Comment (szarnekow) [reply xtext-website/documentation/350_continuous_integration.md:310]: mwe

Comment (cdietrich) [reply xtext-website/documentation/350_continuous_integration.md:310]: EMF is 2.36 which is correct 
 https://github.com/eclipse-simrel/simrel.build/pull/92
 had already preliminary adjusted that one.
for mwe did not anticipate new release

**Review by cdietrich — APPROVED** [review]

**Pull request merged**